README.md

```diff
@@ -57,6 +79,15 @@ You can find the latest version on
 ZIO-Spark is compatible with Scala 2.11, 2.12 and 2.13. Spark is provided, you must add your own Spark version in 
 build.sbt (as you would usually). 
 
+```scala
+libraryDependencies ++= Seq(
+  "io.univalence"    %% "zio-spark"  % "0.10.0",
+  "org.apache.spark" %% "spark-core" % "3.3.1" % Provided,
```

Comment:
Are you sure about the Provided flag for the user imports ?

Reply:
yes, that what we should to be compatible with a spark-submit.